lib/fastlane_core/configuration/config_item.rb

```diff
@@ -73,8 +73,6 @@ def auto_convert_value(value)
       # Weird because of https://stackoverflow.com/questions/9537895/using-a-class-object-in-case-statement
 
       case
-      when data_type == Array
-        return value.to_s.split(',')
```

Comment:
How about just to be sure we add

```ruby
when data_type == Array
  return value.split(',') if value.kind_of?(String)
```

Not sure if that makes sense, but e.g. have you tried passing a value using an environment variable?